src/hooks/reviewHooks/useUpdateReviewRequestViewed.ts

```diff
@@ -0,0 +1,19 @@
+import type { AxiosError } from "axios"
+import { useMutation } from "react-query"
+import type { UseMutationResult } from "react-query"
+
+import * as ReviewService from "services/ReviewService"
+
+export const useUpdateReviewRequestViewed = (): UseMutationResult<
+  void,
+  AxiosError<{ message: string }>,
+  { siteName: string; prNumber: number }
+> => {
+  return useMutation<
+    void,
+    AxiosError<{ message: string }>,
+    { siteName: string; prNumber: number }
+  >(({ siteName, prNumber }) =>
+    ReviewService.updateReviewRequestViewed(siteName, prNumber)
```

Comment:
could i clarify why we don't need to invalidate the cache and force a refetch of the related getter?